tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -318,6 +323,37 @@ def _json_from_tf_examples(self, tf_examples):
       json_exs.append(json_ex)
     return json_exs
 
+  def _json_to_tf_examples(self, examples):
+    def add_single_feature(feat, value, ex):
+      if isinstance(value, integer_types):
+        ex.features.feature[feat].int64_list.value.append(value)
+      elif isinstance(value, Number):
+        ex.features.feature[feat].float_list.value.append(value)
+      else:
+        ex.features.feature[feat].bytes_list.value.append(value.encode('utf-8'))
```

Comment:
just double checking if we need py2 py3 protection for value.encode here.

Reply:
verified this works in both py2 and py3